spiel/st/m0t1fs_spiel_sns_common_inc.sh

```diff
@@ -135,24 +134,24 @@ spiel_prepare()
 	echo SPIEL_FIDS_LIST=$SPIEL_FIDS_LIST
 
 	# install "motr" Python module required by m0spiel tool
-	cd $M0_SRC_DIR/utils/spiel
+	cd "$TOPDIR"/utils/spiel
 	python2 setup.py install --record $PYTHON_STUFF > /dev/null ||\
 		die 'Cannot install Python "motr" module'
 	cd -
 }
 
 spiel_cleanup()
 {
-	cd $M0_SRC_DIR/utils/spiel
+	cd "$TOPDIR"/utils/spiel
 	cat $PYTHON_STUFF | xargs rm -rf
 	rm -rf build/ $PYTHON_STUFF
 	cd -
 }
 
 spiel_sns_repair_start()
 {
-echo $M0_SRC_DIR/utils/spiel/m0spiel $SPIEL_OPTS
-    $M0_SRC_DIR/utils/spiel/m0spiel $SPIEL_OPTS <<EOF
+echo "$TOPDIR"/utils/spiel/m0spiel $SPIEL_OPTS
```

Comment:
enclose $SPIEL_OPTS in double quotes. I think that should fix Codacy warning.

Reply:
@mehjoshi , I added the quotes earlier and it was throwing error, because the code in m0spiel file was based on the value of SPIEL_OPTS.
![image](https://user-images.githubusercontent.com/86768616/162016554-92f18af5-10f7-4c01-9f2e-fbd39b46d917.png)

Earlier code
`def address_get():
    i = 0
    for arg in sys.argv:
        i = i+1
        if format(str(arg)) == "-l":
            break`
Here, the for loop was using '-l' parameter from SPIEL_OPTS variable to get the transport layer. This code is changed recently, so we will need to test the ST's again.

Reply:
@swapnil-seagate, I don't understand. Why should enclosing $SPIEL_OPTS at line 153 and 154 give errors? What was the error?